app/javascript/src/components/Dashboard/Notes/utils.js

```diff
@@ -0,0 +1,10 @@
+import dayjs from "dayjs";
+
+var relativeTime = require("dayjs/plugin/relativeTime");
```

Comment:
You seem to have not understood the differences between `var`, `const` and `let`. Don't use `var`!

Read: https://dev.to/mindninjax/stop-using-var-for-declaring-variables-2p3a

Lemme know if you still haven't understood why we should not use `var`.

Reply:
Sorry for that. I have made the necessary change.